centos_6/cloudify-wagon-builder.sh

```diff
@@ -13,10 +14,15 @@ echo "manylinux1_compatible = False" > "/env/bin/_manylinux.py"
 if test -f ${CONSTRAINTS_FILE}
 then
     echo "## $CONSTRAINTS_FILE exist"
-    wagon create -s . -r -v -f -a '--no-cache-dir -c '${CONSTRAINTS_FILE}''
+    wagon create -r ${REQUIRMENTS_FILE} -v -f -a '--no-cache-dir -c '${CONSTRAINTS_FILE}'' .
 else
     echo "## $CONSTRAINTS_FILE doesn't exist"
-    wagon create -s . -r -v -f
+    wagon create -r ${REQUIRMENTS_FILE} -v -f .
 fi
 
+echo "appending platform to wagon file name"
+OS_DIST=$(python -c 'import platform; print(platform.linux_distribution(full_distribution_name=False)[0])')
+OS_RELEASE=$(python -c 'import platform; print(platform.linux_distribution(full_distribution_name=False)[2])')
+ls /packaging/*.wgn | sed 'p;s/\.wgn/\-'${OS_DIST}'-'${OS_RELEASE}'.wgn/' | xargs -n2 mv
+
```

Comment:
Should we really be putting the platform name in the wagon? Also version and all that why not "cloudify-{project}-plugin.wgn".

Reply:
@EarthmanT , well yeah , because if we didn't do that , the file will be overridden when uploaded to s3 since centos/redhat will have the same name, besides the wagon that got created didn't work when build with `0.6.3` [ we checked the wagon wheels against `0.3.2`, and we found that some wheels are missing when it was build using `0.6.3` ] , so until we fix that , `0.6.3` is not good for us :)